client/merchant-feedback-prompt/style.scss

```
@@ -0,0 +1,37 @@
+.wcpay-merchant-feedback-modal {
+	&.components-modal__frame {
+		max-width: 960px;
+		max-height: calc( 100% - 120px );
+
+		@media screen and ( max-width: $break-small ) {
+			margin: auto auto;
+			max-width: 90vw;
+		}
+	}
+
+	.components-modal__header {
+		border-bottom: none;
+
+		.components-modal__header-heading {
+			font-size: 20px;
+			font-weight: 400;
+			line-height: 28px;
+		}
+	}
+
+	.components-modal__content {
+		padding: 0;
+		overflow: hidden;
+	}
+
+	&__iframe {
+		width: 100%;
+		height: 100%;
+		border: none;
+		display: block;
+		@media ( min-width: 600px ) {
+			width: 600px;
+			height: 500px;
+		}
+	}
+}
```

After some tinkering, I think this would suffice, and fixes mobile view.

Ideally, we could remove the internal padding from the Crowdsignal iframe, but instead I've left the `padding: 0` override declared for the modal.

Have a try and let me know what you think.

```suggestion
.wcpay-merchant-feedback-modal {
	.components-modal__content {
		padding: 0;
		overflow: hidden;
	}

	&__iframe {
		width: 100%;
		height: 100%;
		border: none;
		min-height: 530px; // Removes the vertical scrollbar.

		@media ( min-width: 600px ) {
			width: 600px;
		}
	}
}

```